docs/reference/extensibility/NuGet-Cross-Platform-Plugins.md

```diff
@@ -47,11 +47,18 @@ Under this version, the requirements are as follows:
 - Adhere to the negotiated plugin protocol version.
 - Respond to all requests within a reasonable time period.
 - Honor cancellation requests for any in-progress operation.
+- **Starting with NuGet 6.13, executable plugins (including global .NET tools) must follow these requirements:**  
+  - Naming Convention: Must follow the pattern `nuget-plugin-*`.  

```

Comment:
This makes it sound like existing plugins that worked in 6.12 and earlier will no longer work with 6.13 unless they meet this naming convention.